frontend/src/metabase/components/EntityIdCard/EntityIdCard.tsx

```diff
@@ -14,32 +17,46 @@ const EntityIdTitle = () => {
   );
 
   return (
-    <Group spacing="sm">
-      {t`Entity ID`}
-      <Popover position="top-start">
-        <Popover.Target>
-          <Icon tabIndex={0} name="info" className={Styles.InfoIcon} />
-        </Popover.Target>
-        <Popover.Dropdown>
-          <Paper p="md" maw="13rem">
-            <Text size="sm">
-              {t`When using serialization, replace the sequential ID with this global entity ID to have stable URLs across environments. Also useful when troubleshooting serialization.`}{" "}
-              {showMetabaseLinks && (
-                <>
-                  <Link
-                    target="_new"
-                    to={docsLink}
-                    style={{ color: "var(--mb-color-brand)" }}
-                  >
-                    Learn more
-                  </Link>
-                </>
-              )}
-            </Text>
-          </Paper>
-        </Popover.Dropdown>
-      </Popover>
-    </Group>
+    <SidesheetCardTitle mb={0}>
+      <Group spacing="sm">
+        {t`Entity ID`}
+        <Popover position="top-start">
+          <Popover.Target>
+            <Icon tabIndex={0} name="info" className={Styles.InfoIcon} />
+          </Popover.Target>
+          <Popover.Dropdown>
+            <Paper p="md" maw="13rem">
+              <Text size="sm">
+                {t`When using serialization, replace the sequential ID with this global entity ID to have stable URLs across environments. Also useful when troubleshooting serialization.`}{" "}
+                {showMetabaseLinks && (
+                  <>
+                    <Link
+                      target="_new"
+                      to={docsLink}
+                      style={{ color: "var(--mb-color-brand)" }}
+                    >
+                      Learn more
+                    </Link>
+                  </>
+                )}
+              </Text>
+            </Paper>
+          </Popover.Dropdown>
+        </Popover>
+      </Group>
+    </SidesheetCardTitle>
+  );
+};
+
+export const EntityIdDisplay = ({ entityId }: { entityId: string }) => {
```

Comment:
This would be more reusable if it accepted all the props that Stack accepts as well, and spread those props on line 53